cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DFGPass.kt

```diff
@@ -306,6 +308,11 @@ class DFGPass(ctx: TranslationContext) : ComponentPass(ctx) {
                 node.variable.variables.lastOrNull()?.prevDFGEdges += iterable
             }
         }
+        assume(
+            AssumptionType.AmbiguityAssumption,
+            "If this is not the case, we assume that the last VariableDeclaration in the statement is the one we care about.",
```

Comment:
Not sure if this is a good example. What's an ambiguity assumption? The ambiguity is there, that's not an assumption. The assumption would be that because in 90 % of the cases we only have one variable here, we assume that only one variable is enough to process, or something like that.

Reply:
The types of assumptions we have flexible and can be adapted to your needs, and what seems adequate naming in the moment.